tox.ini

```
@@ -111,17 +111,13 @@ deps =
 [testenv:docs]
 description = Build docs
 commands =
-    python -m sphinx \
-      -a -n -W \
-      --keep-going \
-      -b html --color \
-      -d "{toxinidir}/docs/docstree" \
-      docs/ "{toxinidir}/docs/html"
-    #python -m sphinx_autobuild docs/ "{toxinidir}/docs/html"
+  python -m sphinx \
+    -a -n -W \
+    --keep-going \
+    -b html --color \
```

Since you're already changing the args in this PR:
```suggestion
    -b html {tty:--color} \
```